configs/theta/job_script.theta.bash

```diff
@@ -1,78 +0,0 @@
-#!/bin/bash
-# Copyright (c) 2017,  Los Alamos National Security, LLC (LANS)
-# and the University Corporation for Atmospheric Research (UCAR).
-#
-# Unless noted otherwise source code is licensed under the BSD license.
-# Additional copyright and license information can be found in the LICENSE file
-# distributed with this code, or at http://mpas-dev.github.com/license.html
-#
-#COBALT -t 1:00:00
-#COBALT -n 1
-#COBALT -A OceanClimate
-#COBALT -q debug-flat-quad
-
-export OMP_NUM_THREADS=1
-
-module unload python
-module use /projects/OceanClimate/modulefiles/all
-module load e3sm-unified/1.1.2
```

Comment:
@xylar: shouldn't we substitute the lines above with:
```
source /lus/theta-fs0/projects/ClimateEnergy_2/software/e3sm_unified/base/etc/profile.d/conda.sh
conda activate e3sm_unified_1.2.0_py2.7_nox
```

Reply:
Ah, I didn't realize Theta also used COBALT, which makes it simpler.

I imagine you'd also need to remove the line
```
#COBALT -q debug-flat-quad
```

Does Coooley support `aprun`?  If not, that would also need to be changed (probably just setting `command_prefix=""` or something like that).

If you want to test that out, feel free.

Reply:
since I am testing a-prime now, I will do some of this testing over there first.